src/Infrastructure/MatomoBannerSubmissionTracker.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Infrastructure;
+
+class MatomoBannerSubmissionTracker implements BannerSubmissionTracker {
+
+	private \MatomoTracker $matomoTracker;
+
+	public function __construct( \MatomoTracker $matomoTracker ) {
+		$this->matomoTracker = $matomoTracker;
+	}
+
+	public function track( string $campaign, string $keyword ): void {
+		$this->matomoTracker->setAttributionInfo( json_encode( [ $campaign, $keyword ] ) )
```

Comment:
Thinking about it, I think the Matomo call should go into a try/catch and we try log if it fails or something. Eg if the tracker goes down and this code crashes the user won't be redirected to the payment service.